File: lang/java/test/fun/vac/util/TestRunnerTest.java
import fun.vac.util.TestRunner;

public class TestRunnerTest {

    public static void testParseTest() {
        for (int i = 0; i < 10; i++) {
            if ((i & 1) == 0) {
                TestRunner.parseTest(false);
            } else {
                TestRunner.parseTest(true);
            }
        }
    }

    public static void main(String[] args) {
        testParseTest();
    }
}

File: lang/python/test/fun/vac/util/TestRunnerTest.py
from fun.vac.util import TestRunner

def testParseTest() -> None:
    for i in range(10):
        if (i & 1) == 0:
            TestRunner.parseTest(False)
        else:
            TestRunner.parseTest(True)

def main() -> None:
    testParseTest()

if __name__ == "__main__":
    main()
